services/uwsgi.ini

```diff
@@ -16,9 +18,26 @@ chmod-socket = 666
 logto = /var/log/uwsgi/%n.log
 log-format = %(addr) - - [%(ltime)] "%(method) %(uri) %(proto)" %(status) %(size) "%(referer)" "%(uagent)" host_hdr=%(host) req_time_elapsed=%(msecs)
 
-processes = 6
+; max number of workers; %k is the number of detected CPU cores
+processes = %(%k + 1)
 threads = 2
```

Comment:
@holingpoon from what you've seen for a lot of these network-based apps, do you think this might be more efficient if it's ran on one thread rather than 2?